src/SA/uptime/entry.c

```diff
@@ -0,0 +1,49 @@
+#include <windows.h>
+#include "bofdefs.h"
+#include "base.c"
+
+
+void printUptime() {
+
+	// Counts millisecond ticks since last boot
+	ULONGLONG ticks = KERNEL32$GetTickCount64();
+
+	ULONGLONG seconds = ticks/1000;
```

Comment:
division / modulus of ULONGLONG is not supported on x86 under BOF using the mingw compiler.

either modify the makefile to only compile x64, and modify the cna to print an error if someone attempts to run on a 32 bit beacon, or investigate an alternative approach for x86.
<img width="1027" alt="Screen Shot 2021-01-11 at 11 17 43 AM" src="https://user-images.githubusercontent.com/3172440/104215720-c95c4c80-53fe-11eb-81d5-b14c4b615a11.png">

Reply:
Good thinking. GetTickCount() works fine(ish) although the arithmetic to calculate boot time screws up for some reason. For now I've just pulled that functionality out.

Reply:
works for me, I may tweak it in the future if I find a way to handle the 64bit values in 32bit bof

Reply:
Please do!